frontend/src/App.css

```
@@ -1,36 +1,42 @@
   display: flex;
   height: 100vh;
   width: 100vw;
   overflow: hidden;
-  background: var(--color-cream);
+  background: var(--bg);
   position: relative;
 }
 
-/* Elegant page entrance animation */
+/* Staggered entrance */
 @keyframes fadeSlideIn {
```

Stylelint (17.13.0)

[error] Expected keyframe name "fadeSlideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)